### DISCUSSION TITLE
Connecting MinIO to Tooljet

### DISCUSSION THREAD

**Discussion by Gunkev** [ToolJet/ToolJet#3321] [Q&A]
Hello I want to connect tooljet to MinIO. I ran min.exe server in my Windows OS everything seems to working well. I was able to connect and create my buckets. But when I try to connect to Tooljet I get the error invalid port:9001. My console is running on http://127.0.0.1:9001/
![errorMin](https://user-images.githubusercontent.com/36812576/174478084-0b82ce39-59d7-41c9-a60a-c32db8a0b62b.PNG)

**Comment by sherfin94** [comment]
Hey @Gunkev , I don't have much familiarity with Minio but Google tells me that Minio default port for incoming connections is `9000`  ( and for console is `9001`), could it be that you should be trying to connect to `9000`?

**Reply by Gunkev** [reply]
> Hey @Gunkev , I don't have much familiarity with Minio but Google tells me that Minio default port for incoming connections is `9000` ( and for console is `9001`), could it be that you should be trying to connect to `9000`?

@sherfin94 it returns the same error with port 9000

**Reply by sherfin94** [reply]
I see, @gsmithun4 do you have any suggestions?

**Reply by gsmithun4** [reply]
@sherfin94 Not able to find any

**Reply by gsmithun4** [reply]
@Gunkev can you please try host as `localhost`?

**Reply by sherfin94** [reply]
Might also be a good idea to turn off `ssl` and try @Gunkev

**Reply by Gunkev** [reply]
> @Gunkev can you please try host as `localhost`?

yes I tried localhost but the error persist

**Comment by Gunkev** [comment]
Yeah have try turning off ssl but it stills show the error
